app.js

```diff
@@ -109,6 +125,7 @@ function handleClicks(e) {
   }
 }
 
+// removes 3 images after 25 clicks
 function removePhotos() {
   leftEl.src = null;
```

Comment:
You should set this to be an empty string instead of `null` to avoid errors in the HTML.